tools/cldr-apps/js/src/esm/cldrTable.js

```diff
@@ -754,41 +754,19 @@ function updateRowCodeCell(tr, theRow, cell) {
  */
 function updateRowEnglishComparisonCell(tr, theRow, cell) {
   if (theRow.displayName) {
-    const hintPos = theRow.displayName.indexOf("[translation hint");
-    let hasExample = false;
-    if (theRow.displayExample) {
-      hasExample = true;
-    }
-    if (hintPos != -1) {
-      theRow.displayExample =
-        theRow.displayName.substr(hintPos, theRow.displayName.length) +
-        (theRow.displayExample
-          ? theRow.displayExample.replace(/\[translation hint.*?\]/g, "")
-          : "");
-      theRow.displayName = theRow.displayName.substr(0, hintPos);
-    }
     cell.appendChild(
       cldrDom.createChunk(theRow.displayName, "span", "subSpan")
     );
-    const TRANS_HINT_ID = "en_ZZ"; // must match SurveyMain.TRANS_HINT_ID
+    const TRANS_HINT_ID = "en"; // expected to match SurveyMain.TRANS_HINT_ID
```

Comment:
probably ideally this should go back to being the 'baseline language' wherever that is, should be `en`… and delete this constant

Reply:
I've always wondered what affect "en_ZZ" actually had on the browser in this context... and now I wonder whether "en" does anything; might the comparison column otherwise display right-to-left or something depending on the locale?